gno.land/pkg/gnoweb/frontend/js/realmhelp.ts

```diff
@@ -45,34 +45,43 @@ class Help {
     this.funcList = this.DOM.funcs.map((funcEl) => new HelpFunc(funcEl));
 
     this.restoreAddress();
+    this.restoreMode();
     this.bindEvents();
   }
 
-  private restoreAddress(): void {
-    const { addressInput } = this.DOM;
-    if (addressInput) {
-      const storedAddress = localStorage.getItem("helpAddressInput");
-      if (storedAddress) {
-        addressInput.value = storedAddress;
-        this.funcList.forEach((func) => func.updateAddr(storedAddress));
+  private restoreValue(storageKey: string, inputElement: HTMLInputElement | HTMLSelectElement | null, updateCallback: null | ((value: string) => void)): void {
```

Comment:
maybe updateCallback don't need to be null ?
```suggestion
  private restoreValue(storageKey: string, inputElement: HTMLInputElement | HTMLSelectElement | null, updateCallback: ((value: string) => void)): void {
```

Reply:
I think it could be `null` because a `restore` doesn't necessarily have to update something else than itself, but since we only have callbacks example here, why not.